Implement Button Component

hyeonjin6530 opened the pull request:
전역변수 색상을 사용한 Button Component 제작

hyeonjin6530 committed:
Implement Button Component

hyeonseung-lee commented:
잘했네

VarGun commented on src/component/Button.js:
1. children -> text or btnText 로 대체
2.  disabled 랑 cancel 스타일링 부분 위치 서로 바꾸기 -> disabled 의 적용 우선 순위가 높아짐 -> cancel 버튼도 disabled 처리를 할 일이 있으므로

VarGun replied:
주석 처리 매우 훌륭 .....

hyeonjin6530 committed:
Fix Button.js
Changes to be committed:
modified:   src/component/Button.js
1. props 이름 변경:  children -> btnText
2. cancel과 disabled의 스타일링 위치 변경

VarGun reviewed (approved):
최고 !

hyeonseung-lee commented:
머지 안받아여?

hyeonjin6530 committed:
fix Button.js
On branch front-end-10
Changes to be committed: -> component 폴더 이름 변경
	renamed:    src/component/Button.js -> src/components/Button.js

Changes not staged for commit: -> props 이름  btnText을 children으로
변경
	modified:   src/App.js
        modified:   src/components/Button.js

hyeonjin6530 committed:
fix Button.js
 On branch front-end-10

 Changes to be committed:
	modified:   src/App.js
	modified:   src/components/Button.js
	-> props btnText의 이름을 children으로 변경

+) component 폴더명을 components로 변경

VarGun commented:
추가 수정사항 있어서 일러뒀습니당

hyeonseung-lee commented:
잘하자 연진아
